src/test/record/records_service/test_response_time.py

```diff
@@ -134,39 +102,20 @@ def test_cross_flow_case(self):
         records = create_records(records_raw, columns)
         response = ResponseTime(records)
 
-        expect_raw = [
-            {'start': 0, 'end': 4},
-            {'start': 3, 'end': 4},
-            {'start': 3, 'end': 6},
-            {'start': 6, 'end': 6},
-        ]
-        result = to_dict(response.to_records())
-        assert result == expect_raw
-
         expect_raw = [
             {'start_min': 0, 'start_max': 3, 'end': 4},
             {'start_min': 3, 'start_max': 6, 'end': 6},
         ]
-        result = to_dict(response.to_response_records())
+        result = to_dict(response.to_stacked_bar())
         assert result == expect_raw
 
         expect_raw = [
-            {'start': 0, 'end': 4},
-            {'start': 0, 'end': 6},
-            {'start': 0, 'end': 8},
-            {'start': 0, 'end': 10},
             {'start': 3, 'end': 4},
-            {'start': 4, 'end': 8},
             {'start': 6, 'end': 6},
         ]
-        result = to_dict(response.to_records(all_pattern=True))
+        result = to_dict(response.to_best_case_stacked_bar())
         assert result == expect_raw
```

Comment:
Is it intentional that 'start' : 4 is not taken into account?

Reply:
The `cross_case` may be different from the desired specification. (The implement may wrong.)

However, as this PR is a refactoring, this investigation and correction will not be conducted.